test/Dockerfile.centos7

```diff
@@ -0,0 +1,32 @@
+FROM centos:7
+
+# Install packages
+RUN yum install -y centos-release-scl-rh epel-release sudo make gcc openssl-devel llvm-toolset-7-clang llvm5.0
+
+# Create postgres user/group with specific IDs
+ARG UID=1000
+ARG GID=1000
+
+RUN groupadd -g $GID -o postgres
+RUN useradd -m -u $UID -g $GID -o -s /bin/bash postgres
+
+# Add PostgreSQL repository
+RUN rpm --import http://yum.postgresql.org/RPM-GPG-KEY-PGDG
+RUN rpm -ivh https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+
+# Install PostgreSQL
+ENV PGVERSION=14
```

Comment:
Does it mean we always use PG14 regardless of branches where a pull request is created? I think it won't work on other branches.